src/main/java/tak/TakServer.java

```diff
@@ -6,32 +6,49 @@
 package tak;
 
 import java.io.IOException;
+import java.net.InetSocketAddress;
 import java.net.ServerSocket;
 import java.net.Socket;
 import java.net.SocketTimeoutException;
 import java.util.Date;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import com.sun.net.httpserver.HttpServer;
+
+import tak.httpHandlers.AddSeekHandler;
+import tak.httpHandlers.GetSeeksHandler;
+
 /**
  *
  * @author chaitu
  */
 public class TakServer extends Thread{
 	public static int port;
 	public static int portws;
+	public static int portHttp;
```

Comment:
Something to note about the ports is that nginx is doing a reverse proxy to the app to map the ports so we'll want to make sure it's defined in the properties and notated in the readme.

Reply:
✔️  I'm already reading the port from `properties.xml`.

We also need to think about the authentication of whoever uses this API. For a first run we could just not expose it and expect the PNT backend to run on the same host, so that it can use the loopback address to query the playtak API.